src/frontends/tensorflow/src/decoder_argdef.cpp

```diff
@@ -0,0 +1,86 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "decoder_argdef.hpp"
+
+#include "op_def.pb.h"
+#include "openvino/frontend/tensorflow/node_context.hpp"
+#include "openvino/frontend/tensorflow/special_types.hpp"
+#include "types.pb.h"
+
+namespace ov {
+namespace frontend {
+namespace tensorflow {
+
+namespace {
+const std::map<::tensorflow::DataType, ov::element::Type>& TYPE_MAP() {
```

Comment:
I don't see where you use this map in the file, is it needed here?

Reply:
It is used below in `get_attribute` method

Reply:
Ah, okay. But this map was copied from another file, can we have it only in one place?

Reply:
I would like to leave Decoder implementation to be standalone to avoid any conflicts with Protobuf versions, you should remember. OVTF can forget or can happen something different that this TYPE_MAP will be got for other protobuf version.